diff --git a/Makefile b/Makefile
index 704580ee..9472cb6f 100644
--- a/Makefile
+++ b/Makefile
@@ -106,7 +106,7 @@ electron-test:
 		-s $(PLATFORM)
 
 assets/dmg/background.tiff: assets/dmg/background.png assets/dmg/background@2x.png
-	tiffutil -cathidpicheck $^ -out $@
+	#tiffutil -cathidpicheck $^ -out $@
 
 electron-build: assets/dmg/background.tiff | $(BUILD_TEMPORARY_DIRECTORY)
 	$(RESIN_SCRIPTS)/electron/build.sh \
